### PR TITLE
Upgraded config for automatic publishing

### PR DESCRIPTION
**Pull request by plehegar**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/pull/98.html" title="Last updated on Oct 1, 2020, 12:23 PM UTC (af8af70)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/98/9b17e2b...af8af70.html" title="Last updated on Oct 1, 2020, 12:23 PM UTC (af8af70)">Diff</a>

### CODE REVIEW

**Comment by plehegar** [comment]
Note that neither algorithm nor data-algorithm are documented https://respec.org/docs/ .

**Review by yoavweiss — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   'remove'></script>
   <script class='remove'>
   var respecConfig = {
-    shortName: "hr-time",
+    shortName: "hr-time-3",
```
Comment (yoavweiss) [inline index.html:10]: Would it be possible to keep the `hr-time` name pointing to the ED and have `hr-time-2` pointing to the L2 REC version?

Comment (plehegar) [reply index.html:10]: yes. already the case in fact. w3.org/TR points to L3 and w3.org/TR/hr-time-2 points to the REC. Respec doesn't allow you (yet) to use a version number in the dated link but not in the latest version link.

**Pull request merged**